pydantic/types.py

```diff
@@ -1008,3 +1012,42 @@ def to(self, unit: str) -> float:
             raise errors.InvalidByteSizeUnit(unit=unit)
 
         return self / unit_div
+
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ DATE TYPES ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+class PastDate(date):
+    @classmethod
+    def __modify_schema__(cls, field_schema: Dict[str, Any]) -> None:
+        field_schema.update(format='past-date')
```

Comment:
I think this should be removed since `'past-date'` [doesn't seem to be valid](https://json-schema.org/understanding-json-schema/reference/string.html#dates-and-times).
If you remove both `__modify_schema__`, `format: 'date'` will be added thanks to inheritance

Reply:
Ah! Got it! Thank you.

Reply:
Fixed.